run.py

```diff
@@ -73,4 +74,15 @@
 
 # TODO: Add other feature extractions here
 
+# TODO: Run models 
+
+
+# Run our highlight models
+print("Running our Models")
+features_df = df[['pitch', 'amplitude', 'subjectivity', 'polarity', 'pred_excitement']]
+print("Running Random Forest..")
+rf_predictions = runModels.getRandomForestPredictions(features_df)
+print("Running Neural Network..")
+nn_predictions = runModels.getNeuralNetworkPredictions(features_df)
+
```

Comment:
add these columns to the big boi dataframe

Reply:
Fixed